django/chat/views.py

```diff
@@ -369,10 +375,10 @@ def chat_message(request, chat_id):
     user_message_text = request.POST.get("user-message", "").strip()
     mode = chat.options.mode
 
-    logger.info(
+    logger.debug(
         "User message received.",
         chat_id=chat_id,
-        user_message=user_message_text,
+        user_message=f"{user_message_text[:100]}{'...' if len(user_message_text) > 100 else ''}",
```

Comment:
Logger was printing the entire message which could get really crazy if you were entering a whole document for summarization etc.